core/graph/src/main/java/uk/gov/gchq/gaffer/operation/export/graph/OtherGraphExporter.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2016-2017 Crown Copyright
```

Comment:
Same comment as above regarding copyright headers - if this is a new file, it should have 2017 as the copyright year.